package.json

```diff
@@ -23,7 +23,8 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "create-location-data": "node src/scripts/uploadAutoCompleteLocations.js",
-    "get-location-data": "node src/scripts/getAutoCompleteLocations.js"
+    "get-location-data": "node src/scripts/getAutoCompleteLocations.js",
+    "build-glean": "glean translate src/telemetry/metrics.yaml -f javascript -o src/telemetry/generated"
```

Comment:
Are there long-form versions of `-f` and `-o`? If so, I'd prefer we use those so future readers have a better idea of what they do :)